### ISSUE TITLE
Stuck on ktfmt settings

### ISSUE THREAD

**Issue by smarteist** [facebook/ktfmt#504]
There is bug, maybe in dropdown section here is the stacktrace:

```
2024-08-13 05:50:47,541 [  44028] SEVERE - #c.i.o.a.i.FlushQueue - Cannot invoke "Object.equals(Object)" because the return value of "javax.swing.JComboBox.getSelectedItem()" is null
java.lang.NullPointerException: Cannot invoke "Object.equals(Object)" because the return value of "javax.swing.JComboBox.getSelectedItem()" is null
        at com.facebook.ktfmt.intellij.KtfmtConfigurable.isModified(KtfmtConfigurable.java:105)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.isModified(ConfigurableWrapper.java:186)
        at com.intellij.openapi.options.newEditor.SettingsEditor.checkModifiedInternal(SettingsEditor.java:522)
        at com.intellij.openapi.options.newEditor.SettingsEditor.checkModifiedForItem(SettingsEditor.java:516)
        at com.intellij.openapi.options.newEditor.SettingsEditor.checkModified(SettingsEditor.java:504)
        at com.intellij.openapi.options.newEditor.SettingsEditor$5.updateCurrent(SettingsEditor.java:229)
        at com.intellij.openapi.options.newEditor.ConfigurableEditor.lambda$select$2(ConfigurableEditor.java:250)
        at com.intellij.openapi.util.ExecutionCallback.setExecuted(ExecutionCallback.java:44)
        at com.intellij.openapi.util.ActionCallback.setDone(ActionCallback.java:72)
        at com.intellij.ui.CardLayoutPanel.select(CardLayoutPanel.java:121)
        at com.intellij.ui.CardLayoutPanel.lambda$selectLater$0(CardLayoutPanel.java:132)
        at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:27)
        at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
        at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
        at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
        at com.intellij.openapi.application.impl.RwLockHolder.runIntendedWriteActionOnCurrentThread(RwLockHolder.kt:204)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:830)
        at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:419)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:138)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:129)
        at com.intellij.openapi.application.impl.ApplicationImpl.runWithImplicitRead(ApplicationImpl.java:1152)
        at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:81)
        at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:123)
        at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:43)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
        at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
        at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
        at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:699)
        at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$12(IdeEventQueue.kt:593)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithoutImplicitRead(RwLockHolder.kt:105)
        at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:593)
        at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:77)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:362)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:361)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:843)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:361)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:356)
        at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1022)
        at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
        at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1022)
        at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:356)
        at com.intellij.openapi.application.impl.RwLockHolder.runIntendedWriteActionOnCurrentThread(RwLockHolder.kt:204)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:830)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:398)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:121)
        at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
        at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
        at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
        at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
        at java.desktop/java.awt.Dialog.lambda$show$2(Dialog.java:1081)
        at java.desktop/sun.awt.SunToolkit.performOnMainThreadIfNeeded(SunToolkit.java:2170)
        at java.desktop/java.awt.Dialog.show(Dialog.java:1041)
        at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:898)
        at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:474)
        at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1754)
        at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1703)
        at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:104)
        at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:61)
        at com.intellij.ide.actions.ShowSettingsAction.actionPerformed(ShowSettingsAction.java:48)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.kt:304)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks$lambda$4(ActionUtil.kt:275)
        at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.performWithActionCallbacks(ActionManagerImpl.kt:1170)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.kt:274)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction$lambda$5(ActionMenuItem.kt:269)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:227)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction(ActionMenuItem.kt:260)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem._init_$lambda$0(ActionMenuItem.kt:71)
        at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed$lambda$4(ActionMenuItem.kt:102)
        at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
        at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.kt:101)
        at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:525)
        at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:558)
        at java.desktop/java.awt.Component.processMouseEvent(Component.java:6657)
        at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
        at java.desktop/java.awt.Component.processEvent(Component.java:6422)
        at java.desktop/java.awt.Container.processEvent(Container.java:2266)
        at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
        at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4969)
        at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4583)
        at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4524)
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
        at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2809)
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:794)
        at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
        at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:766)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:763)
        at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:699)
        at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:638)
        at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:584)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:138)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:129)
        at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
        at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:77)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:362)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:361)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:843)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:361)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:356)
        at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1022)
        at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
        at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1022)
        at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:356)
        at com.intellij.openapi.application.impl.RwLockHolder.runIntendedWriteActionOnCurrentThread(RwLockHolder.kt:209)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:830)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:398)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
2024-08-13 05:50:47,553 [  44040] SEVERE - #c.i.o.a.i.FlushQueue - Android Studio Koala | 2024.1.1 Patch 2  Build #AI-241.18034.62.2411.12169540
2024-08-13 05:50:47,554 [  44041] SEVERE - #c.i.o.a.i.FlushQueue - JDK: 17.0.11; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-08-13 05:50:47,554 [  44041] SEVERE - #c.i.o.a.i.FlushQueue - OS: Linux
2024-08-13 05:50:47,554 [  44041] SEVERE - #c.i.o.a.i.FlushQueue - Plugin to blame: ktfmt version: 1.1.0.46
2024-08-13 05:50:47,554 [  44041] SEVERE - #c.i.o.a.i.FlushQueue - Last Action: ShowSettings
```

Same Error with newer verions:

```
2024-08-13 06:36:13,361 [  22039] SEVERE - #c.i.u.c.QueueProcessor - Cannot invoke "Object.equals(Object)" because the return value of "javax.swing.JComboBox.getSelectedItem()" is null
java.lang.NullPointerException: Cannot invoke "Object.equals(Object)" because the return value of "javax.swing.JComboBox.getSelectedItem()" is null
        at com.facebook.ktfmt.intellij.KtfmtConfigurable.isModified(KtfmtConfigurable.java:105)
        at com.intellij.openapi.options.ex.ConfigurableWrapper.isModified(ConfigurableWrapper.java:186)
        at com.intellij.openapi.options.newEditor.SettingsEditor.checkModifiedInternal(SettingsEditor.java:522)
        at com.intellij.openapi.options.newEditor.SettingsEditor.checkModifiedForItem(SettingsEditor.java:516)
        at com.intellij.openapi.options.newEditor.SettingsEditor.checkModified(SettingsEditor.java:504)
        at com.intellij.openapi.options.newEditor.SettingsEditor$5.updateCurrent(SettingsEditor.java:229)
        at com.intellij.openapi.options.newEditor.ConfigurableEditor.updateIfCurrent(ConfigurableEditor.java:238)
        at com.intellij.openapi.options.newEditor.ConfigurableEditor$4.run(ConfigurableEditor.java:195)
        at com.intellij.util.ui.update.ContextAwareUpdate.run$lambda$1$lambda$0(ContextAwareUpdate.kt:49)
        at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
        at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
        at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:86)
        at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:81)
        at com.intellij.util.ui.update.ContextAwareUpdate.run(ContextAwareUpdate.kt:48)
        at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:354)
        at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:344)
        at com.intellij.util.ui.update.MergingUpdateQueue.doFlush(MergingUpdateQueue.java:301)
        at com.intellij.util.ui.update.MergingUpdateQueue.lambda$flush$1(MergingUpdateQueue.java:280)
        at com.intellij.util.ui.EdtInvocationManager.invokeAndWaitIfNeeded(EdtInvocationManager.java:79)
        at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:280)
        at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:250)
        at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
        at com.intellij.util.concurrency.ChildContext$runAsCoroutine$1.invoke(propagation.kt:81)
        at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:86)
        at com.intellij.util.concurrency.ChildContext.runAsCoroutine(propagation.kt:81)
        at com.intellij.util.Alarm$Request.lambda$runSafely$0(Alarm.java:369)
        at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:253)
        at com.intellij.util.Alarm$Request.runSafely(Alarm.java:369)
        at com.intellij.util.Alarm$Request.run(Alarm.java:356)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
        at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:272)
        at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
        at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
        at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
        at com.intellij.openapi.application.impl.RwLockHolder.runIntendedWriteActionOnCurrentThread(RwLockHolder.kt:204)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:830)
        at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:419)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:138)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:129)
        at com.intellij.openapi.application.impl.ApplicationImpl.runWithImplicitRead(ApplicationImpl.java:1152)
        at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:81)
        at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:123)
        at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:43)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
        at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:699)
        at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$12(IdeEventQueue.kt:593)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithoutImplicitRead(RwLockHolder.kt:105)
        at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:593)
        at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:77)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:362)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:361)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:843)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:361)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:356)
        at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1022)
        at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
        at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1022)
        at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:356)
        at com.intellij.openapi.application.impl.RwLockHolder.runIntendedWriteActionOnCurrentThread(RwLockHolder.kt:204)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:830)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:398)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:121)
        at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
        at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
        at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
        at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
        at java.desktop/java.awt.Dialog.show(Dialog.java:1079)
        at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:898)
        at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:474)
        at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1754)
        at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1703)
        at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:104)
        at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:61)
        at com.intellij.ide.actions.ShowSettingsAction.actionPerformed(ShowSettingsAction.java:48)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.kt:304)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks$lambda$4(ActionUtil.kt:275)
        at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.performWithActionCallbacks(ActionManagerImpl.kt:1170)
        at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.kt:274)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction$lambda$5(ActionMenuItem.kt:269)
        at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:227)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction(ActionMenuItem.kt:260)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem._init_$lambda$0(ActionMenuItem.kt:71)
        at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed$lambda$4(ActionMenuItem.kt:102)
        at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:106)
        at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
        at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.kt:101)
        at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:525)
        at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:558)
        at java.desktop/java.awt.Component.processMouseEvent(Component.java:6662)
        at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3394)
        at java.desktop/java.awt.Component.processEvent(Component.java:6427)
        at java.desktop/java.awt.Container.processEvent(Container.java:2266)
        at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5032)
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
        at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4963)
        at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4577)
        at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4518)
        at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
        at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2811)
        at java.desktop/java.awt.Component.dispatchEvent(Component.java:4860)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
        at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
        at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
        at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:699)
        at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:638)
        at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:584)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:138)
        at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:129)
        at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:584)
        at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:77)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:362)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:361)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:843)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:361)
        at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:356)
        at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1022)
        at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
        at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1022)
        at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:356)
        at com.intellij.openapi.application.impl.RwLockHolder.runIntendedWriteActionOnCurrentThread(RwLockHolder.kt:209)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:830)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:398)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
2024-08-13 06:36:13,374 [  22052] SEVERE - #c.i.u.c.QueueProcessor - Android Studio Koala | 2024.1.1 Patch 2  Build #AI-241.18034.62.2411.12169540
2024-08-13 06:36:13,375 [  22053] SEVERE - #c.i.u.c.QueueProcessor - JDK: 21.0.2; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o.
2024-08-13 06:36:13,375 [  22053] SEVERE - #c.i.u.c.QueueProcessor - OS: Linux
2024-08-13 06:36:13,376 [  22054] SEVERE - #c.i.u.c.QueueProcessor - Plugin to blame: ktfmt version: 1.1.0.51
2024-08-13 06:36:13,376 [  22054] SEVERE - #c.i.u.c.QueueProcessor - Last Action: ShowSettings
```

**Comment by hick209** [comment]
Possibly the same issue as https://github.com/facebook/ktfmt/issues/494, which was fixed by https://github.com/facebook/ktfmt/pull/501

Please retry this on 0.52 once we release later this week (or with changes from master) and reopen the issue in case this has not been fixed

**Comment by smarteist** [comment]
@hick209 

> Possibly the same issue as #494, which was fixed by #501
> 
> Please retry this on 0.52 once we release later this week (or with changes from master) and reopen the issue in case this has not been fixed

@hick209 

Thank you, I confirm that it is fixed in 0.52.
